Translation corrections (#220)
07th-mod/higurashi-console-arcs: Update/kage_001.txt

//rそれは、暁自身から固く口止めされていたという建前上の理由もあったけど、…私が二人に課した、ちょっとした『試験』でもあったりする。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それは、暁自身から固く口止めされていたという建前上の理由もあったけど、…私が二人に課した、ちょっとした『試験』でもあったりする。",

> 			NULL, "Part of it was because Akira himself swore us to silence... but I also saw it as as a little bit of a \"trial\" imposed on the two of them.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Part of it was because Akira himself swore us to silence... but I also saw it as a little bit of a \"trial\" imposed on the two of them.", Line_Normal);
```

//r夏美はそう言って、にこやかに笑う。…まぁ、私たちにとっては今さら慣れっこの進路面談も彼女には一大事件だったのだから、クリアして一安心といったところだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "夏美はそう言って、にこやかに笑う。…まぁ、私たちにとっては今さら慣れっこの進路面談も彼女には一大事件だったのだから、クリアして一安心といったところだろう。",

> 			NULL, "Natsumi smiled while saying that. ...Well, the career interview was a big deal for her, so she must be so relieved to have it behind herr.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Natsumi smiled while saying that. ...Well, the career interview was a big deal for her, so she must be so relieved to have it behind her.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s08/37/320300034", 256, TRUE);
	OutputLine(NULL, "「でしょでしょ？",
			NULL, "\"I know, right?", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s08/37/320300035", 256, TRUE);
	OutputLine(NULL, "　しかも狙いは新設の学部だから、難易度はそこそこのレベルだし。",
			NULL, " Plus, you're aiming for the newly established department, so the difficulty level seems reasonable.", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s08/37/320300036", 256, TRUE);
	OutputLine(NULL, "しかも空調完備で学業にはもってこいの環境！」",

> 			NULL, " And not only that, but it's fully packed with air conditioning, making it a perfect environment for studying!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " And not only that, but it's fully air conditioned, making it a perfect environment for studying!\"", Line_Normal);
```
